### PR TITLE
Update categories.py for pdmv

### PR DESCRIPTION
**Pull request by perrotta**
Add Adriano and Mikko to the pdmv category

### CODE REVIEW

**Comment by perrotta** [comment]
+1

File: categories.py [hunk 1]
```diff
@@ -52,6 +52,7 @@
   "clacaputo":        ["reconstruction"],
   "kskovpen":         ["pdmv"],
   "sunilUIET":        ["pdmv"],
+  "miquork":          ["pdmv"],
```
Comment (smuzaffar) [inline categories.py:55]: @perrotta , `miquork` is already defined at https://github.com/cms-sw/cms-bot/blob/bc8a9cd1465a2d7128a18a09edc7066ac3de9ddd/categories.py#L121 , please remove it from here and add `"pdmv"` at https://github.com/cms-sw/cms-bot/blob/bc8a9cd1465a2d7128a18a09edc7066ac3de9ddd/categories.py#L121

File: categories.py [hunk 2]
```diff
@@ -65,7 +66,7 @@
   "aloeliger":        ["l1"],
   "rvenditti":        ["dqm"],
   "syuvivida":        ["dqm"],
-  "tjavaid":	      ["dqm"],
+  "tjavaid":	        ["dqm"],
```
Comment (smuzaffar) [inline categories.py:69]: can you please remove the extra `space` from here?

File: categories.py (hunk 1 above)
Comment (perrotta) [reply categories.py:55]: thank you, I did not notice!

**Comment by perrotta** [comment]
Moved to https://github.com/cms-sw/cms-bot/pull/2033

**Pull request closed without merging**